### PR TITLE
Update Notify Telegram configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
A new PR since my old one went to the wrong branch 😢 

Update style of notify telegram documentation to follow new configuration variables description.
Related to #6385.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/notify.telegram.markdown [hunk 1]
```diff
@@ -84,10 +84,16 @@ notify:
...
-- **chat_id** (*Required*): The chat ID of your user.
+{% configuration %}
+name:
+  description: Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
+  required: false
+  type: string
+chat_id:
+  description: The chat ID of your user.
+  required: true
+  type: string
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: This got to be an integer.

File: source/_components/notify.telegram.markdown [hunk 2]
```diff
@@ -84,10 +84,16 @@ notify:
     chat_id: CHAT_ID_2
 ```
 
-Configuration variables:
-
-- **name** (*Optional*): Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
-- **chat_id** (*Required*): The chat ID of your user.
+{% configuration %}
+name:
+  description: Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: The default value is mentioned in the description.

File: source/_components/notify.telegram.markdown [hunk 3]
```diff
@@ -133,15 +151,36 @@ action:
...
+username:
+  description: Username for a URL which require HTTP authentication.
+  required: false
+  type: string
+password:
+  description: Password for a URL which require HTTP authentication.
+  required: false
+  type: string
+authentication:
+  description: Set to 'digest' to use HTTP digest authentication, defaults to 'basic'.
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: Default

File: source/_components/notify.telegram.markdown [hunk 4]
```diff
@@ -176,15 +215,36 @@ action:
...
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: Default

File: source/_components/notify.telegram.markdown [hunk 5]
```diff
@@ -204,15 +264,36 @@ action:
...
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: Default

File: source/_components/notify.telegram.markdown [hunk 6]
```diff
@@ -230,9 +311,21 @@ action:
...
-
-- **latitude** (*Required*): The latitude to send.
-- **longitude** (*Required*): The longitude to send.
-- **keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom keyboard.
-- **inline_keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data.
+{% configuration %}
+latitude:
+  description: The latitude to send.
+  required: true
+  type: [string, int]
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: I believe it has to be `float`. But it is not yet supported.

File: source/_components/notify.telegram.markdown [hunk 7]
```diff
@@ -230,9 +311,21 @@ action:
...
+longitude:
+  description: The longitude to send.
+  required: true
+  type: [string, int]
```
Comment (WofWca) [inline source/_components/notify.telegram.markdown]: Same

**Review by WofWca — CHANGES_REQUESTED** [review]

File: source/_components/notify.telegram.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: Must I put `notify` as default or just notify without ` `?

File: source/_components/notify.telegram.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: integer or int? in the documentation they mention "int"

File: source/_components/notify.telegram.markdown (hunk 3 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: Done

File: source/_components/notify.telegram.markdown (hunk 6 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: I thought so, but what do you have to fill in then? I think for now it has to be a string.

File: source/_components/notify.telegram.markdown (hunk 1 above)
Comment (WofWca) [reply source/_components/notify.telegram.markdown]: See #6452. Last documentation changes haven't gone live yet, apparently.

File: source/_components/notify.telegram.markdown (hunk 2 above)
Comment (WofWca) [reply source/_components/notify.telegram.markdown]: Not sure actually. But if you're going to use ``` `notify` ```, be aware that you'll have to quote it.

**Comment by klaasnicolaas** [comment]
Have the file updated on 3 comments after yet

File: source/_components/notify.telegram.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: Have it left out, maybe that @frenck could give more clarity.

File: source/_components/notify.telegram.markdown (hunk 6 above)
Comment (frenck) [reply source/_components/notify.telegram.markdown]: @klaasnicolaas Please use `float`, I'm going to add it to the list of allowed types for the documentation standards.

File: source/_components/notify.telegram.markdown (hunk 7 above)
Comment (frenck) [reply source/_components/notify.telegram.markdown]: @klaasnicolaas Please use `float`, I'm going to add it to the list of allowed types for the documentation standards.

File: source/_components/notify.telegram.markdown [hunk 8]
```diff
@@ -133,15 +152,37 @@ action:
...
-
-- **url** or **file** (*Required*): For local or remote path to an image.
-- **caption** (*Optional*): The title of the image.
-- **username** (*Optional*): Username for a URL which require HTTP authentication.
-- **password** (*Optional*): Username for a URL which require HTTP authentication.
-- **authentication** (*Optional*): Set to 'digest' to use HTTP digest authentication, defaults to 'basic'.
-- **keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom keyboard.
-- **inline_keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data.
+{% configuration %}
+url or file:
```
Comment (frenck) [inline source/_components/notify.telegram.markdown]: This feels weird... 2 fields in 1...

**Review by frenck — CHANGES_REQUESTED** [review]
Awaiting changes

File: source/_components/notify.telegram.markdown (hunk 8 above)
Comment (klaasnicolaas) [reply source/_components/notify.telegram.markdown]: I think it must by url then?

**Comment by frenck** [comment]
@klaasnicolaas `url` and `file` occur in multiple places. 
I think the best option is to list them both as separate options to the configuration, mark them both required and add a note on how on how one of them is required.

**Comment by klaasnicolaas** [comment]
@frenck I will adjust it, I do not really know how I could best formulate that notite so that the other person understands it.

**Comment by frenck** [comment]
@klaasnicolaas I've added a suggestion. What do you think?

**Comment by klaasnicolaas** [comment]
Seems fine to me 😉

**Comment by klaasnicolaas** [comment]
@frenck Seems fine to me 😉

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks again @klaasnicolaas! 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**